language/stdlib/modules/validator_set.mvir

```diff
@@ -1,30 +1,93 @@
 module ValidatorSet {
+    import 0x0.LibraAccount;
     import 0x0.Vector;
 
-    struct TrustedPeer {
-        account_address: address,
+    struct ValidatorInfo {
+        addr: address,
+        network_signing_pubkey: bytearray,
+        network_identity_pubkey: bytearray,
+        consensus_pubkey: bytearray,
+        // TODO: voting power
+    }
+
+    // A current or prospective validator should publish one of these under their address
+    resource ValidatorConfig {
```

Comment:
what's the difference between the `Config` and `Info` and `resource` vs `struct`?